static/docker-compose.yml

```diff
@@ -27,17 +27,32 @@ services:
     - plextracdb
     - redis
     image: "plextrac/plextracapi:${UPGRADE_STRATEGY:-stable}"
+    volumes:
+    - uploads:/usr/src/plextrac-api/uploads:rw
```

Comment:
This is needed because we are writing the ETL full output json to the `uploads/etl-logs` folder so we can find detailed info if we need to troubleshoot anything during the rollout. This can go away after that.